packages/consent/docs/README.md

```diff
@@ -158,57 +235,45 @@ with_defaults views: [:department, :small_volumes] do
 end
 ```
 
-## Permission
+### Permission
 
-A permission is what is consented to the user. It consentment to perform
-an *action* on a limited *view* of the *subject*. It marries the three concepts
-to consent an access to the user.
+Permission is what is granted to a role, or a user. It grants the ability to perform an *action*,
+on a limited scope (*view*) of the *subject*.
 
 ## CanCan Integration
 
-Consent provides a CanCan ability (Consent::Ability) to integrate your
-permissions with frameworks like Rails. To use it with Rails check out the
-example at [Ability for Other Users](https://github.com/CanCanCommunity/cancancan/wiki/Ability-for-Other-Users)
-on CanCanCan's wiki.
-
-In the ability you define the scope of the permissions. This is typically a
-user:
+Consent provides a CanCan ability (Consent::Ability) that can be initialized with a group of
```

Comment:
In case people are needing extra context:

```suggestion
Consent provides a [CanCan](https://github.com/CanCanCommunity/cancancan#readme) ability (Consent::Ability) that can be initialized with a group of
```